### ISSUE TITLE
No cursor showing up, active selected element is not highlighted

### ISSUE THREAD

**Issue by bastianwegge** [jesseduffield/lazygit#1414]
**Describe the bug**
I cannot see the selected element in my terminal. I have attached a screenshot where the cursor should highlight the uppermost element of the the "files" tab, but it doesnt.

**Expected behavior**
Lazygit should start up and show the currently selected element. When I switch in between files for example, the window of files should show the selected element (or active cursor) respectively.

**Screenshots**
![screenshot_of_no_cursor](https://user-images.githubusercontent.com/1541474/128309753-a1c31d0e-e036-4d0a-babb-62679e620aaf.png)

**Desktop (please complete the following information):**
 - OS: macOS BigSur 11.5.1
 - Lazygit Version 0.28.2

**Additional context**
I tried to figure out if it's a `gocui`-Issue, installed `go` and `jsonui` (using `go get -u github.com/gulyasm/jsonui`) as a reference but I can see the cursor in `jsonui`. As I am using `homebrew` for package-management I thought maybe it's an issue caused by the binary build, so I installed `lazygit` using `go get github.com/jesseduffield/lazygit` but nothing changed. It is worth mentioning that this bug also occurs for me on lazydocker.

I'd also like to add what an incredible productivity boost lazygit means for me. Thank you for building and maintaining this!

**Comment by jesseduffield** [comment]
@bastianwegge is this a recent issue or has your terminal always had this problem with lazygit? At any rate let me know if this helps you: https://github.com/jesseduffield/lazygit/blob/master/docs/Config.md#struggling-to-see-selected-line

**Comment by bastianwegge** [comment]
@jesseduffield thanks for the quick response! This seems like a recent issue, but sadly I cannot really determine when it was introduced. I've been using `lazygit` without an actual cursor for the last 2-3 months.

I found that the `#struggling-to-see-selected-line` had a section of "maybe you disabled bold fonts" which I found out you have to enable when using `iTerm2`. It has an option stating `Text Rendering: [ ] Draw bold Text in bold font`. With that option enabled now the selection starts to differentiate from the other elements in the list slightly, but enough to figure out which item is selected. I am using `Hack Nerd Font Mono` by the way.

I guess we can close this issue for now. `jsonui` has this principle of showing a cursor-selection with a highlighted background. I tried to get that to work with the `lazygit` config parameters, but that didn't work, even with the `config` option provided `lazygit --config=~/Library/Application\ Support/lazygit/config.yml`. Maybe there's something missing on my end, but the theme should show the selected line with a red background-color, right?

My theme:

```yml
  theme:
    lightTheme: false # For terminals with a light background
    activeBorderColor:
      - white
      - bold
    inactiveBorderColor:
      - white
    optionsTextColor:
      - blue
    selectedLineBgColor:
      - red
    selectedRangeBgColor:
      - red
```

**Comment by MaiLunJiye** [comment]
**Feature request :**  cursor character


```
https://github.com/jesseduffield/lazygit/blob/master/docs/Config.md#struggling-to-see-selected-line
```

Although the configuration will work in most of time, I hope lazygit can add a cursor character such as ">" to mark the selected item. It is ultimate guarantee If the environment does not support color, bold .etc.


for example： we can clearly identify the selected item without color or bold.

```
┌─Commits - Reflog────────────────────────────────────────┐
│  4930d19e OA 6.2.6 Redis 6.2.6                          │
│  aba95175 OA corrupt-dump-fuzzer test, avoid creating ju│
│> 7540708a su Fix missing check for sanitize_dump in corr│
│  664c7b36 YP Allow to override OPENSSL_PREFIX (#9567)   │
│  c390972c YG Propagate OPENSSL_PREFIX to hiredis. (#9345│
│  73d286d5 OA Fix stream sanitization for non-int first v│
│  4cbccab2 DC TLS build fix on OpenBSD when built with Li│
│  23ed37dd yv Fix: client pause uses an old timeout (#947│
└────────────────────────────────────────────────1 of 300─┘

```
